Dockerfile

```diff
@@ -22,11 +22,11 @@ RUN wget -q https://github.com/AsamK/signal-cli/releases/download/v${SIGNAL_CLI_
     ln -sf /opt/signal-cli-${SIGNAL_CLI_VERSION}/bin/signal-cli /usr/local/bin/
 
 WORKDIR /app
-COPY scripts/register-number.sh /app
+COPY scripts/register-number.sh .
 COPY requirements.txt .
+COPY setup.cfg .
+COPY setup.py .
```

Comment:
Are these files necessary? Also I think it might be best (albeit redundant) to copy the `signal_scanner_bot` folder into the container in addition to having it mounted by the compose file. That way if we ever wanted to build/publish/ship the container directly it would have all of the code present on it. The mounting just allows us to restart the containers without having to rebuild the container.